frontend/core-ui/src/modules/automations/components/builder/edges/primary.tsx

```
@@ -1,34 +1,30 @@
-  sourceX,
-  sourceY,
-  targetX,
-  targetY,
-  sourcePosition,
-  targetPosition,
-  selected,
-  source,
-  sourceHandleId,
-  data,
-}) => {
+const Edge: FC<EdgeProps> = (edge) => {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Edge'. Consider to delete it or rename it.

'Edge' is defined here:


(lint/suspicious/noRedeclare)